client/components/tinymce/index.jsx

```diff
@@ -161,29 +162,29 @@ module.exports = React.createClass( {
 	displayName: 'TinyMCE',
 
 	propTypes: {
-		mode: React.PropTypes.string,
-		onActivate: React.PropTypes.func,
-		onBlur: React.PropTypes.func,
-		onChange: React.PropTypes.func,
-		onDeactivate: React.PropTypes.func,
-		onFocus: React.PropTypes.func,
-		onHide: React.PropTypes.func,
-		onInit: React.PropTypes.func,
-		onRedo: React.PropTypes.func,
-		onRemove: React.PropTypes.func,
-		onReset: React.PropTypes.func,
-		onShow: React.PropTypes.func,
-		onSubmit: React.PropTypes.func,
-		onUndo: React.PropTypes.func,
-		onSetContent: React.PropTypes.func,
-		tabIndex: React.PropTypes.number,
-		isNew: React.PropTypes.bool,
-		onTextEditorChange: React.PropTypes.func,
-		onKeyUp: React.PropTypes.func
+		mode: PropTypes.string,
+		onActivate: PropTypes.func,
+		onBlur: PropTypes.func,
+		onChange: PropTypes.func,
+		onDeactivate: PropTypes.func,
+		onFocus: PropTypes.func,
+		onHide: PropTypes.func,
+		onInit: PropTypes.func,
+		onRedo: PropTypes.func,
+		onRemove: PropTypes.func,
+		onReset: PropTypes.func,
+		onShow: PropTypes.func,
+		onSubmit: PropTypes.func,
+		onUndo: PropTypes.func,
+		onSetContent: PropTypes.func,
+		tabIndex: PropTypes.number,
```

Comment:
The EVENTS array contains the `init` and `mouseUp` events, should we add them here as well?

I tend to prefer grouping all the event handlers together at the end, being the first props all non-functions. That way I can have a quick feeling for the kind of events the component supports by just looking at the eventHandlers properties.

Reply:
It looks like `onInit` is already there, but I'll add `onMouseOver` and re-organize the handlers at the bottom. Thanks :)

Reply:
Oh, sorry for the confusion, I meant, `onInput` not `onInit`!

Reply:
Ah, thanks, I'll add that as well.